build/Targets/Settings.props

```diff
@@ -26,6 +26,7 @@
     <JenkinsBuild Condition="'$(JENKINS_HOME)' != ''">true</JenkinsBuild>
     <DeveloperBuild>false</DeveloperBuild>
     <DeveloperBuild Condition="'$(OfficialBuild)' != 'true' AND '$(JenkinsBuild)' != 'true'">true</DeveloperBuild>
+    <ContinuousIntegrationBuild Condition="!$(DeveloperBuild)">true</ContinuousIntegrationBuild>
```

Comment:
This will be treu during an official build. That doesn't seem right.

Reply:
Hmm. I guess I always think of CI as Jenkins only and Microbuild as something else. I'll adjust my model. 

---
In reply to: [196531381](https://github.com/dotnet/roslyn/pull/26981#discussion_r196531381) [](ancestors = 196531381)